README.md

```diff
@@ -0,0 +1,93 @@
+# drupaldiversity.github.io
+Public website for the Drupal Diversity project
+
+# DrupalDiversity.github.io Contributor's Guide
+
+When adding documents, be sure to use [GitHub Flavored Markdown](https://guides.github.com/features/mastering-markdown/ "GitHub Flavored Markdown").
+
+### Technical Team
+
+- cleverington
+- seanr
+- justachris
+- justafish
+- sugaroverflow
+- zombree
+
```

Comment:
Not sure how up to date we need to keep the team list, but @mlhess and @rgoodine are also on the roster